### PR TITLE
support multiple schema in postgre import

### PR DESCRIPTION
**Pull request by RajkaranSh**
Currenly by default the jdbc connection assumes that the public   schema should be used in case of postgresql.
but in entity automation case the schema could be different than public.
have added the support for that.

### CODE REVIEW

File: sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala [hunk 1]
```diff
@@ -128,8 +129,13 @@ class JDBCImport(
     val dbType = dbutils.secrets.get(scope = databricksScope, key = "DB_TYPE")
 
     val database = importConfig.database
+    val schema = importConfig.schema
 
-    val connectionUrl = s"jdbc:$dbType://$host:$port/$database"
+    var connectionUrl = s"jdbc:$dbType://$host:$port/$database"
+
+    if (dbType == "postgresql" && schema.isDefined){
```
Comment (nisheet195) [inline sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala]: Use constants postgres variable

Comment (RajkaranSh) [reply sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala]: .

Comment (RajkaranSh) [reply sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala]: done

**Review by nisheet195 — APPROVED** [review]
Lgtm

**Pull request merged**